<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="scaleSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="scaleSpin">
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="DavaGLWidget" name="davaGlWidget" native="true"/>
        </item>
        <item>
         <widget class="QScrollBar" name="verticalScrollBar">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="hierarchyDockWidget">
   <property name="windowTitle">
    <string>Hierarchy Tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="HierarchyTreeWidget" name="hierarchyDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="libraryDockWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propertiesWidgetContent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="propertiesWidgetLayout"/>
     </item>
    </layout>
   </widget>
   <widget class="LibraryWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuProject">
     <property name="title">
      <string>Project</string>
     </property>
     <addaction name="actionNew_platform"/>
     <addaction name="actionNew_screen"/>
    </widget>
    <addaction name="actionNew_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionClose_project"/>
    <addaction name="separator"/>
    <addaction name="menuProject"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFontManager"/>
    <addaction name="actionLocalizationManager"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="title">
     <string> Help</string>
    </property>
    <addaction name="actionHelpContents"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_project"/>
   <addaction name="actionOpen_project"/>
   <addaction name="actionSave_project"/>
   <addaction name="actionClose_project"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="propertiesDockWidget">
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="PropertyGridContainerWidget" name="propertiesDockWidgetContents"/>
  </widget>
  <action name="actionOpen_project">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/openproject.png</normaloff>:/icons/openproject.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="toolTip">
    <string>Open project</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/newscene.png</normaloff>:/icons/newscene.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionHierarchy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hierarchy tree</string>
   </property>
  </action>
  <action name="actionLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionFontManager">
   <property name="text">
    <string>Font Manager</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/101.png</normaloff>:/icons/101.png</iconset>
   </property>
   <property name="text">
    <string>Close project</string>
   </property>
   <property name="toolTip">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionLocalizationManager">
   <property name="text">
    <string>Localization Manager</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/savescene.png</normaloff>:/icons/savescene.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="toolTip">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="text">
    <string>Help contents</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/118.png</normaloff>:/icons/118.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/117.png</normaloff>:/icons/117.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNew_platform">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/079.png</normaloff>:/icons/079.png</iconset>
   </property>
   <property name="text">
    <string>New platform</string>
   </property>
  </action>
  <action name="actionNew_screen">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/icons/068.png</normaloff>:/icons/068.png</iconset>
   </property>
   <property name="text">
    <string>New screen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HierarchyTreeWidget</class>
   <extends>QWidget</extends>
   <header>HierarchyTreeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DavaGLWidget</class>
   <extends>QWidget</extends>
   <header>davaglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryWidget</class>
   <extends>QWidget</extends>
   <header>librarywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PropertyGridContainerWidget</class>
   <extends>QWidget</extends>
   <header>propertygridcontainerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
